libs/components/indicators/src/lib/modules/tokens/token.component.html

```diff
@@ -1,8 +1,8 @@
 <div
   class="sky-token sky-btn sky-btn-default"
+  [attr.disabled]="disabled ? 'true' : null"
```

Comment:
Is this valid HTML? From the spec it doesn't look like a DIV has a `disabled` attribute. https://www.w3.org/TR/html4/index/attributes.html

Could you use `aria-disabled=true` instead?